packages/express/src/__tests__/acceptance/test-helpers.ts

```diff
@@ -0,0 +1,129 @@
+// Copyright IBM Corp. 2020. All Rights Reserved.
+// Node module: @loopback/express
+// This file is licensed under the MIT License.
+// License text available at https://opensource.org/licenses/MIT
+
+import {
+  Binding,
+  createProxyWithInterceptors,
+  intercept,
+  InterceptorOrKey,
+} from '@loopback/context';
+import {Client, givenHttpServerConfig, supertest} from '@loopback/testlab';
+import bodyParser from 'body-parser';
+import {ExpressApplication} from '../../express.application';
+import {SpyAction, SpyConfig} from '../fixtures/spy-config';
+import spyFactory from '../fixtures/spy.middleware';
+export const spy = spyFactory;
+export {SpyConfig} from '../fixtures/spy-config';
+
+export type TestFunction = (spyBinding: Binding<unknown>) => Promise<unknown>;
+
+export class TestHelper {
+  readonly app: ExpressApplication;
+  client: Client;
+
+  constructor() {
+    this.app = new ExpressApplication({
+      express: {
+        ...givenHttpServerConfig(),
+        settings: {
+          env: 'test',
+        },
+      },
+    });
+    this.app.expressServer.expressMiddleware(
+      bodyParser.json,
+      {strict: false},
+      {key: 'middleware.bodyParser'},
+    );
+  }
+
+  async start() {
+    await this.app.start();
+    this.client = supertest(this.app.expressServer.url);
+  }
+
+  stop() {
+    return this.app.stop();
+  }
+
+  bindController(interceptor?: InterceptorOrKey) {
+    const interceptors: InterceptorOrKey[] = [];
+    if (interceptor) interceptors.push(interceptor);
+    class MyController {
+      @intercept(...interceptors)
+      hello(msg: string) {
+        return `Hello, ${msg}`;
+      }
+    }
+    const binding = this.app.controller(MyController);
+    this.app.expressServer.expressApp.post('/hello', async (req, res, next) => {
```

Comment:
a bit confused: the testing endpoint is set up by express server, why not use the REST decorators? Is there a difference?

Reply:
This module does not have dependency on `@loopback/rest` by design. It should possible to use it with plain Express.

Reply:
Any advantage we have, by this being an independent package ?